vendor/vendor.json

```diff
@@ -1,5 +1,10 @@
 {
 	"comment": "",
+	"heroku": {
```

Comment:
Why?

Reply:
That was auto generated, do we even want to keep using govendor?

Reply:
Also that was previously at the bottom, I have no idea why govendor decided to move it up to here.

Reply:
We probably should... but we don't need the heroku stuff.